.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/unifiedpush-admin-ui/issues{}) 
+
+## What
+Add a short answer for: What was done in this PR? (E.g Don't allow users has access to the feature X.)
+
+## Why
+Add a short answer for: Why it was done? (E.g The feature X was deprecated.)
+
+## How
+Add a short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 
+
+## Verification Steps
```

Comment:
I think "Verification steps", "checklist", and "progress" could all just be "Verification Steps".